tapalcatl_server/handler.go

```diff
@@ -116,6 +121,14 @@ func logBool(x bool) string {
 	}
 }
 
+func convertNanosToMillis(nanoseconds int64) int64 {
+	return nanoseconds / 1000000
```

Comment:
Would this be more readable using the constant `time.Millisecond` rather than `1000000`? That seems to be [recommended by the docs](https://golang.org/pkg/time/#Duration).